lib/std/Build/Step/Compile.zig

```diff
@@ -1848,14 +1829,11 @@ fn make(step: *Step, prog_node: *std.Progress.Node) !void {
     }
 
     for (self.framework_dirs.items) |directory_source| {
-        if (b.sysroot != null) {
-            try zig_args.append("-iframeworkwithsysroot");
-        } else {
-            try zig_args.append("-iframework");
-        }
-        try zig_args.append(directory_source.getPath2(b, step));
+        const path = directory_source.getPath(b);
```

Comment:
`getPath2` (how it was before) is more helpful when there are missing step dependencies.

Reply:
Ah, I didn't know, lemme revert this change. Thanks!